gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -534,10 +534,7 @@ gum_module_entry_from_path_and_base (const gchar * path,
 
   module = gum_elf_module_new_from_memory (path, base_address, NULL);
 
-  if (module == NULL ||
-      dwarf_elf_init_b (gum_elf_module_get_elf (module), DW_DLC_READ,
-          DW_GROUPNUMBER_ANY, gum_on_dwarf_error, NULL, &dbg, &error)
```

Comment:
Hmm, it looks like `entry->dbg` is always `NULL` with this change. Is that intentional?